docs/rebuild_admin.rst

```diff
@@ -0,0 +1,407 @@
+However it may have been named differently during the installation of your 
+instance. You can use the command ``getent passwd 1000`` to check the username
+corresponding to UID 1000. Once you have the correct username, reset its password
+using the `passwd` command, for example:
+
+.. code:: sh 
+
+ passwd sdadmin
+
+.. important::
+ Make sure to select a strong password, and record it in the *Admin Workstation's*
+ KeePassX database.
+
+Finally, reboot the *Monitor Server* and verify that you can log in at the console
+using the new password.
+
+Repeat the process for the *Application Server*. Use the same username and 
+password as for the *Monitor Server* - this is required in order for the 
+``./securedrop-admin install`` command to work correctly.
+
+Step 3: Set up Admin Workstation access
+=======================================
+Next, you'll configure the servers to allow temporary SSH access from the new *Admin
+Workstation*.
+
+First, start the new *Admin Workstation* with persistence enabled and an administration
+password set.
+
+Next, connect the new *Admin Workstation* to the *Hardware Firewall* via the
+appropriate Ethernet port, and set up its static IP address. For more information
+on how to do so, see :ref:`this section in the firewall setup documentation
+<assign_static_ip_to_workstation>`. If you do not know the correct static IP
+address for the *Admin Workstation*, log in to the *Hardware Firewall's* admin
+interface and check the settings under ``Aliases``.
+
+Next, determine whether your instance was set up to allow adminstrative access
+via SSH over Tor, or via SSH over LAN. If you don't know which option was originally
+chosen, you can check as follows:
+
+ #. Log in to the *Application Server* via the console using the adminstration username
+    and password.
+ #. Check to see if a file ``/var/lib/services/tor/ssh/hostname`` exists. If it
+    does, your instance is set up to use SSH over Tor and you should configure 
+    temporary SSH access using :ref:`these instructions <rebuild_ssh_over_tor>`.
+    If not, your instance is set up to use SSH over LAN, and you should follow
+    :ref:`these instructions instead <rebuild_ssh_over_lan>`.
+
+.. _rebuild_ssh_over_tor:
+
+Configuring access for an SSH-over-Tor instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Direct SSH access is disabled when the SSH-over-Tor option is selected during
+installation. To temporarily re-enable it, you'll need to update ``iptables`` rules
+and change the sshd daemon's configuration.
+
+First, log on to the *Application Server* via the console, and run the following
+commands, substituting the *Admin Workstation's* static IP for ``<admin_static_ip>``:
+
+.. code:: sh
+
+  sudo iptables -I INPUT -p tcp -dport 22 -s <admin_static_ip> \
+    -m state --state NEW,ESTABLISHED -j ACCEPT
+  sudo iptables -I OUTPUT -p tcp -sport 22 -m state --state ESTABLISHED -j ACCEPT 
```

Comment:
Both `iptables` commands fail with "Bad argument '22'". The culprit appears to be `-sport`, `-dport` (should be `--sport`, `--dport`).

Reply:
fixed